lib/beaker/dsl/install_utils.rb

```diff
@@ -1052,6 +1053,12 @@ def install_pe
       #       for Unix like systems and puppet-enterprise-VERSION.msi for Windows systems.
       # @api dsl
       def upgrade_pe path=nil
+        set_console_password = false
+        # if we are upgrading from something lower than 3.4 then we need to set the pe console password
+        if (dashboard[:pe_ver] ? version_is_less(dashboard[:pe_ver], "3.4.0") : true)
+          set_console_password = true
+        end
```

Comment:
isn't this just an overly complicated way of writing the statement below?

    set_console_password = !dashboard[:pe_ver] || version_is_less(dashboard[:pe_ver], '3.4.0')